sources/packages/forms/Dockerfile

```diff
@@ -32,13 +30,12 @@ ENV NPM_PACKAGES=/.npm-packages \
 RUN echo "prefix = $NPM_PACKAGES" >> ~/.npmrc
 
 # Include details of the required dependencies
-COPY ./package.json $NPM_PACKAGES/
-COPY ./package-lock.json $NPM_PACKAGES/
-
-# Use "Continuous Integration" to install as-is from package-lock.json
-RUN npm ci --prefix=$NPM_PACKAGES
+RUN mkdir -p $NPM_PACKAGES/
+# Include details of the required dependencies
+RUN cp package.json $NPM_PACKAGES/
 
-RUN apk del git
+# Use install as-is from package.json
+RUN npm i --prefix=$NPM_PACKAGES
```

Comment:
What is the benefit of npm i over npm ci here. 

Formio docker file uses npm ci, thats why asked.

Reply:
the package-lock and package.json has mismatched version of libraries, so when the build happens, the installations was not happening due to the conflict.